### DISCUSSION TITLE
Error 429

### DISCUSSION THREAD

**Discussion by ryosoftware** [simple-login/app#1908] [Q&A]
Hello,
Trying to create the aliases I've created before (manually) then the site blocks it after a while.
First block is no more than 5 aliases per minute, but now I'm blocked for more than 10 minutes.
What time is the block time? I need to create more than 100 aliases...

**Comment by nguyenkims** [comment]
@robertoprubio to avoid abuses, a user can't create too many aliases in a short period of time. If you need to transfer aliases, you can import them which isn't limited by the anti abuse measure.

**Comment by braxchat** [comment]
Hello, I’m going through the same problem right now. I didn’t know that there was a threshold. What are the parameters so that I can stay within the non-abuse limits? I need 80 aliases but got stopped at 50 I think.

**Comment by roadrunnerml** [comment]
I'm running into this error, I've run into it a few times, and I paused, then created more aliases.. Is there a max number of aliases we can create? I was informed there wasn't one, but if there is, I need to know. This is critical to my testing team.
Thanks!
